dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -71,7 +70,7 @@ async def _update_stubs(self, change):
             node_id, node_info = change.new
             address = "{}:{}".format(node_info["nodeManagerAddress"],
                                      int(node_info["nodeManagerPort"]))
-            channel = aiogrpc.insecure_channel(address)
+            channel = dashboard_utils.create_insecure_channel(address)
```

Comment:
It's cleaner to set an option here instead of providing a special channel. If the existing API cannot meet our requirements, then we create a new API.
```python
options = (("grpc.enable_http_proxy", 0), )
channel = dashboard_utils.create_insecure_channel(address, options=options)
```

Reply:
I am OK to change it back. And `create_insecure_channel ` is just a wrapper function.